Document chore procedure in README (#308)
wmde/new-lexeme-special-page: README.md

```sh
npm run cypress:open
```

### Chore: Dependency Updates

You can see which dependencies have new releases by first making sure your local dependencies are up-to-date by executing `npm ci` and then running `npm outdated`.
The following dependencies should be ignored:

- Wikit (i.e. `@wmde/wikit-tokens` and `@wmde/wikit-vue-components`):
  we’re using a newer pre-release version and don’t want to downgrade to the latest full release.
- Vue and Vuex:

>   in production, we use the versions shipped by MediaWiki core,

Reviewer: Do we have a place that ought to show what version is shipped by MediaWiki core? If so, we may want to link that here

Reviewer 2: +1 Sounds like a good idea!!

Author: We could link to [foreign-resources.yaml](https://gerrit.wikimedia.org/g/mediawiki/core/+/master/resources/lib/foreign-resources.yaml), but I’m not sure how helpful it is (might be a bit long / confusing)?

Reviewer: Maybe with something like "The current core version is listed in [foreign-resources.yaml](https://gerrit.wikimedia.org/g/mediawiki/core/+/master/resources/lib/foreign-resources.yaml)."
Then it is implied that one has to look through a list of things.

Author: Done, thanks!

>   in production, we use the versions shipped by MediaWiki core,

Reviewer 2: Interesting, didn't know this, but it makes sense ofc.

  so we should use the same versions for testing.

All other dependencies should generally be updated to the latest version.

> If you discover that a dependency cannot be updated for some reason, please add it to the above list.

Reviewer: I think it would be better to emphasize that this is a judgement call, not a limitation:

```suggestion
If you discover that a dependency should not be updated for some reason, please add it to the above list.
```

If a dependency can only be updated with substantial manual work,
you can create a new task for it and skip it in the context of the current chore.

> The recommended way to update dependencies is to collect related dependency updates into grouped commit.

Reviewer: typo:

```suggestion
The recommended way to update dependencies is to collect related dependency updates into grouped commits.
```

Reviewer: Also, a sentence outlining _why_ this is the recommended way would be good. Mentioning probably at least reviewability and debuggability. Though I don't have a good suggestion right now, this could also happen in the next iteration on this section.

Author: Done

Reviewer: The added sentences read like a good start. Thanks!

> This can be one commit for each of:

Reviewer: I would like to weaken this list a bit, as many variations of that might make sense.

```suggestion
For example, this can be one commit for each of:
```

Author: Well, it was “for example” in the task description I based it on, but then I saw that the previous chore seemed to have followed this pattern fairly closely, so it felt strange to only call it an example… but I think that might sound better with the other change (to explain why this is the recommended way), so okay.

- all ESLint-related dependency updates
- all Jest-related dependency updates
- all Vue-related dependency updates
- all PostCSS/Stylelint-related dependency updates
- `npm update` for all other dependency updates

Make sure that all checks still pass for every commit.

> Once all dependency updates are merged, update the submodule version in WikibaseLexeme by running `npm run bump-special-new-lexeme` there:

Reviewer 2: Nice add.
